converters/toZigbee.js

```diff
@@ -2853,6 +2853,37 @@ const converters = {
             }
         },
     },
+    tuya_roller_blind_control: {
+        key: ['state', 'position'],
+        convertSet: async (entity, key, value, meta) => {
+            if (key === 'position') {
+                if (value >= 0 && value <= 100) {
+                    const invert = !(meta.mapped.meta && meta.mapped.meta.coverInverted ?
+                        !meta.options.invert_cover : meta.options.invert_cover);
+                    value = invert ? 100 - value : value;
+                    await sendTuyaCommand(entity, 514, 0, [4, 0, 0, 0, value]); // Set position from 0 - 100%
+                } else {
+                    meta.logger.debug('_TZE200_wmcdj3aq: Curtain motor position is out of range');
+                }
+            } else if (key === 'state') {
+                value = value.toLowerCase();
+                switch (value) {
+                case 'close':
+                    await sendTuyaCommand(entity, 1025, 0, [1, 0]); // close
+                    break;
+                case 'open':
+                    await sendTuyaCommand(entity, 1025, 0, [1, 2]); // open
```

Comment:
We can merge this with the existing converter, e.g. change `await sendTuyaCommand(entity, 1025, 0, [1, isTZE200_wmcdj3aq ? 1 : 0]); // 0x04 0x01: Open` and above define `const isTZE200_wmcdj3aq = entity.getDevice().manufacturerName === 'TZE200_wmcdj3aq`).

Reply:
@Koenkk we also considered about your solution at the beginning. By the way, there is a disadvantage of this solution. If there are some new similar models (manufatureName) like 'TZE200_wmcdj3aq' in the future, it is required to update the converter then the code of converter is mess. 

The root of this issue is TS0601 motor has some fingerPrint with opposite way to control the position. So my idea is separate them into 2 sub-categories. In the future, if there is any similar motor/roller bind like this, we just add new fingerPrint to the devices.js only.

Reply:
Hi, @dinhchinh82 Would you please also move model ID "aabybja" and             
{modelID: 'TS0601', manufacturerName: '_TZE200_nkoabg8w'}, to use the new bind method? As they behaves the opposite from the current one.
Thanks very much!

Reply:
@zhouF96 after @Koenkk merge this pull request, it's quite easy to add your fingerPrint device to the devices. js to have fully supported.

Reply:
@dinhchinh82 Thanks very much, and will try it tomorrow morning.